sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs

```diff
@@ -15,5 +15,5 @@
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
 
-[assembly: AssemblyVersion("2.9.0.0")]
-[assembly: AssemblyFileVersion("2.9.0.0")]
+[assembly: AssemblyVersion("2.9.1.0")]
```

Comment:
```suggestion
[assembly: AssemblyVersion("2.9.0.0")]
```
AssemblyVersion only needs to be updated when there's a major version update.

Reply:
We have a small bug fix against the previous one.  We need this update if people installed the old version. @bganapa is that right?

Reply:
Updating `AssemblyVersion` doesn't help the user to upgrade -- user who was using the old version can still build correctly.

Reply:
Here's a great ariticle talking about it: https://support.microsoft.com/en-us/help/556041